### PR TITLE
fix command example for the option --from-env-file

### PR DESCRIPTION
**Pull request by EugeneChung**

### CODE REVIEW

**Comment by Rajakavitha1** [comment]
@Bradamant3  could you please take a look at the updates made at: https://5c6bbf9cc9cead000778cc01--kubernetes-io-vnext-staging.netlify.com/docs/tasks/configure-pod-container/configure-pod-configmap/

**Comment by Bradamant3** [comment]
@EugeneChung Thanks for your PR! I don't see a problem with it -- but I also don't quite see the value add. Adding the comments to the `properties` file seems fine, but why move the content around in the markdown file?

File: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 1]
```diff
@@ -150,20 +150,21 @@ ui.properties:          83 bytes
 
 Use the option `--from-env-file` to create a ConfigMap from an env-file, for example:
 ```shell
+wget https://k8s.io/docs/tasks/configure-pod-container/configmap/kubectl/game-env-file.properties -O configure-pod-container/configmap/kubectl/game-env-file.properties
+cat configure-pod-container/configmap/kubectl/game-env-file.properties
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md]: We could also (in this PR, or separately) make the commands and the sample output separate, so as to match the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output).

**Comment by DanyC97** [comment]
i think this PR can be closed as being fixed by https://github.com/kubernetes/website/pull/13078

**Comment by DanyC97** [comment]
@cody-clark fyi, thanks

**Comment by DanyC97** [comment]
@EugeneChung since #13078 included your changes except the comments in the  `content/en/docs/tasks/configure-pod-container/configmap/kubectl/game-env-file.properties` file i'm going to close this PR unless you planning to rebase this PR ?

would appreciate if you can let us know your intentions in the next few days, thank you

**Comment by EugeneChung** [comment]
Okay. Let me close.

**Pull request closed without merging**